### PR TITLE
fix/TR-3280/header-logo-support-rtl

### PR DESCRIPTION
**Pull request by marpesia**
**Related to:** [TR-3280](https://oat-sa.atlassian.net/browse/TR-3280)

**Description:**
On RTL mode the logo on the Header keeps on the left. There is a CSS class that forces a `float: left`.
I've tracked the CSS class but I don't know the reason to have it. In order to avoid regression, I just disable the floating effect on RTL mode.

**Changes:**

- Add CSS rule to disable `float: left` effect on RTL mode.

**How to check:**

- It is failing only when any user is logged but Tao is on an RTL language. To change Tao language, go to `config/generis.conf.php` and change to `define('DEFAULT_ANONYMOUS_INTERFACE_LANG','ar-arb');` on line 47

**TAE Environment:**

- Link to TAE and credentials in the ticket https://oat-sa.atlassian.net/browse/TR-3280?focusedCommentId=167192

**Local test:**

- Link to Composer in the ticket https://oat-sa.atlassian.net/browse/TR-3280?focusedCommentId=167192

### CODE REVIEW

File: views/package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
         "@oat-sa/tao-core-libs": "0.4.4",
         "@oat-sa/tao-core-sdk": "1.16.3",
         "@oat-sa/tao-core-shared-libs": "1.0.3",
-        "@oat-sa/tao-core-ui": "1.47.0",
+        "@oat-sa/tao-core-ui": "github:oat-sa/tao-core-ui-fe#fix/TR-3280/header-logo-support-rtl",
```
Comment (bziondik) [inline views/package.json:20]: Just reminder, to update after lib will be released

**Review by bziondik — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [x]  Feature is working correctly on playground (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

**Review by krampstudio — CHANGES_REQUESTED** [review]
I'd suggest you to reset the `clear` property either in [views/scss/layout.scss](https://github.com/oat-sa/tao-core/blob/master/views/scss/layout.scss) or maybe in the [views/scss/inc/_header-footer.scss](https://github.com/oat-sa/tao-core/blob/master/views/scss/inc/_header-footer.scss#L284)

**Comment by krampstudio** [comment]
ok so the companion PR is not needed anymore, can you please close it and update the description.

**Review by krampstudio — COMMENTED** [review]
We can see quickly the logo on the left and then moving to the right, but I guess it's because the rtl dir is added after the page has loaded? 
 But after the logo is correctly positionned.

- [ ] New code is covered by tests (not applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] <s>New code is not subject to concurrency issues (if applicable)</s>
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by krampstudio — APPROVED** [review]

**Review by bziondik — APPROVED** [review]

**Review by krampstudio — APPROVED** [review]
The update looks good to me.

**Pull request merged**